bin/kalite

```diff
@@ -78,6 +85,28 @@ if not os.path.exists(kalite_path):
 # Converts to normal string for Windows users, where it gets turned into a unicode string and errors out in subprocess
 env['KALITE_DIR'] = str(kalite_path)
 
+# TODO(benjaoming): Delegate responsibility to the .deb package
+if os.path.isfile("/etc/ka-lite/username"):
```

Comment:
My point is that if this is supposed to be skipped on windows, isn't it more explicit to condition on `os.name` or `sys.platform`? If nothing else, I think it's easy and worthwhile to note the intent in a comment (`# The intent is that this fails on Windows`) . After that I'm happy to merge this.